#!/usr/bin/env python3
PACKAGE = "val_safety_exception_reporter"

from dynamic_reconfigure.parameter_generator_catkin import *

# initialize parameter generator to define parameters
gen = ParameterGenerator()

# add parameters to generator
gen.add("operator_suggestions", bool_t, 0, "Flag to provide suggestions to operator about resolving issues", True)
gen.add("control_level_info",   bool_t, 0, "Flag to provide control-level information about issues",		 True)

# generate necessary files
exit(gen.generate(PACKAGE, "val_safety_exception_reporter", "SafetyExceptionReporterParams"))
